package.json

```diff
@@ -25,7 +25,7 @@
     "csrf": "^3.0.0",
     "cytube-common": "git://github.com/CyTube/cytube-common",
     "cytube-mediaquery": "git://github.com/CyTube/mediaquery",
-    "cytubefilters": "git://github.com/calzoneman/cytubefilters#67c7c69a",
+    "cytubefilters": "git://github.com/davidsiaw/cytubefilters",
```

Comment:
If you've made changes to `cytubefilters` that are necessary to make this work, you should make a PR of those changes in that repo instead of changing this dependency.